java-gradle/Makefile

```diff
@@ -5,15 +5,7 @@ build:
     	#We need to build the build/libs/rookoutDemo-1.0.0.jar - which already include the project sources in the jar
 	gradle fatJar
 
-run:
-	export ROOKOUT_AGENT_HOST=cloud.agent.rookout.com
-	export ROOKOUT_AGENT_PORT=443
-	export ROOKOUT_TOKEN=$(token)
-	java  -javaagent:rook.jar -jar build/libs/rookoutDemo-1.0.0.jar
-
 gradle-run:
-	export ROOKOUT_AGENT_HOST=cloud.agent.rookout.com
-	export ROOKOUT_AGENT_PORT=443
 	export ROOKOUT_TOKEN=$(token)
 	gradle run
```

Comment:
this command will not get ROOKOUT_TOKEN. every command in a Makefile runs in a subshell which do not inherit envs

One solution is to do `export ROOKOUT_TOKEN=$(token) && gradle run`
Other solution is to export token outside of makefile

Reply:
Done